.github/workflows/sanity-test.yaml

```diff
@@ -234,19 +240,19 @@ jobs:
         with:
           service: onedrive
           kind: backup
-          backup-args: '--user "${TEST_USER}"'
-          restore-args: '--folder ${RESTORE_DEST_PFX}${{ steps.new-data-creation-onedrive.outputs.result }} --restore-permissions'
-          test-folder: '${RESTORE_DEST_PFX}${{ steps.new-data-creation-onedrive.outputs.result }}'
+          backup-args: '--user "${{ env.TEST_USER }}"'
+          restore-args: '--folder ${{ env.RESTORE_DEST_PFX }}${{ steps.new-data-creation-onedrive.outputs.result }} --restore-permissions'
+          test-folder: '${{ env.RESTORE_DEST_PFX }}${{ steps.new-data-creation-onedrive.outputs.result }}'
 
       # generate some more enteries for incremental check
       - name: OneDrive - Create new data (for incremental)
         working-directory: ./src/cmd/factory
         run: |
           go run . onedrive files  \
-            --user ${TEST_USER} \
-            --secondaryuser  ${SECONDARY_TEST_USER} \
-            --tenant ${AZURE_TENANT_ID} \
-            --destination ${RESTORE_DEST_PFX}${{ steps.new-data-creation-onedrive.outputs.result }} \
+            --user ${{ env.TEST_USER }} \
+            --secondaryuser  ${{  secrets.CORSO_SECONDARY_M365_TEST_USER_ID }} \
```

Comment:
Primary user coming from `env` and secondary user coming from `secret` looks odd, but I don't really have any strong opinion on this.